staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -442,6 +451,26 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+	
+	mirrorRef, _ := imagereference.Parse(o.To)
```

Comment:
WDYT:
```
func printInstallInstuctions(out io.Writer, from, to, otherrepos []string)
```

be move useful..?

Reply:
This section of the code could continue to evolve in complexity where it could be clearer to break it out into a function.  Will consider this.